.github/workflows/triage.yml

```diff
@@ -16,6 +16,9 @@ jobs:
   triage:
     name: Nissuer
     runs-on: ubuntu-latest
+    if: |
+      (github.event_name != 'issue_comment' && true) ||
```

Comment:
I don't think the `&& true` is necessary, `github.event_name != 'issue_comment'` would already return `true` or `false`